mappings/translation_utilities.go

```diff
@@ -49,6 +49,23 @@ func GetTranslation(tableName models.TableName) (Translation, error) {
 		return PlanDocumentSolutionLinkTranslation()
 	case models.TNPlanDataExchangeApproach:
 		return PlanDataExchangeApproachTranslation()
+	// MTO TABLES
+	case models.TNMTOCategory:
+		return MTOCategoryTranslation()
+	case models.TNMTOMilestone:
+		return MTOMilestoneTranslation()
+	case models.TNMTOSolution:
+		return MTOSolutionTranslation()
+	case models.TNMTOMilestoneSolutionLink:
+		return MTOMilestoneSolutionLinkTranslation()
+	case models.TNMTOInfo:
+		return MTOInfoTranslation()
+	case models.TNMTOCommonSolutionContact:
+		return MTOCommonSolutionContactTranslation()
+	// case models.TNMTOCommonMilestone:
+	// 	return MTOCommonMilestoneTranslation()
+	// case models.TNMTOCommonSolution:
+	// 	return MTOCommonSolutionTranslation()
```

Comment:
Are these still relevant?

Reply:
Thanks! No, I deleted it. They were previously going to be used, but we are trying to use change history without this meta data. We will add back in if needed later.